src/core/io/pla_parser.cpp

```diff
@@ -0,0 +1,68 @@
+#include "core/io/pla_parser.hpp"
+
+namespace syrec {
+
+    [[maybe_unused]] void parse_pla(TruthTable& reader, std::istream& in) {
+        std::string line;
+        std::regex  whitespace("\\s+");
+
+        while (in.good() && getline(in, line)) {
+            trim(line);
+            line = std::regex_replace(line, whitespace, " ");
+            if ((line.empty()) || (line.rfind('#', 0) == 0) || (line.rfind(".i", 0) == 0) || (line.rfind(".o", 0) == 0) || (line.rfind(".ilb", 0) == 0) || (line.rfind(".ob", 0) == 0) || (line.rfind(".p", 0) == 0) || (line.rfind(".type ", 0) == 0)) {
+                continue;
+            }
+
+            else if (line == ".e") {
+                break;
+            }
+
+            else {
+                assert(!line.empty() && (line[0] == '0' || line[0] == '1' || line[0] == '-' || line[0] == '~'));
+
+                std::vector<std::string> inout;
+
+                auto        lineString = std::stringstream(line);
+                std::string tokenString;
+                while (std::getline(lineString, tokenString, ' ')) {
+                    inout.emplace_back(tokenString);
+                }
+
+                if (inout.size() != 2)
+                    std::cerr << "I/O not available" << std::endl;
```

Comment:
This is much worse than the `assert` that was here before. Now the execution continues in the case of an error even in the debug configuration.
Either keep the assert from before or throw an exception here.